artifactory/cli.go

```diff
@@ -809,6 +813,58 @@ func GetCommands() []cli.Command {
 				return replicationDeleteCmd(c)
 			},
 		},
+		{
+			Name:            "permission-target-template",
+			Aliases:         []string{"ptt"},
+			Usage:           permissiontargettemplate.Description,
+			HelpName:        common.CreateUsage("rt ptt", permissiontargettemplate.Description, permissiontargettemplate.Usage),
+			UsageText:       permissiontargettemplate.Arguments,
+			ArgsUsage:       common.CreateEnvVars(),
+			SkipFlagParsing: true,
```

Comment:
This seems to apply to all template commands (repo, permissions and replications) - we should remove the *SkipFlagParsing* option.
As a result, we should also remove the following code section from the beginning of the command functions:
```
	if show, err := showCmdHelpIfNeeded(c); show || err != nil {
		return err
	}
```
We use the *SkipFlagParsing* option when we wish to disable codegangsta's flag parsing (in native commands). There's probably no reason to do this here.